std-bits/base/src/main/java/org/enso/base/numeric/Number_Utils.java

```diff
@@ -0,0 +1,8 @@
+package org.enso.base.numeric;
+
+public class Number_Utils {
+  // For testing only.
+  public static Double floatId(Double d) {
+    return d;
+  }
+}
```

Comment:
If this is only used for tests, it should be moved to `test/Base_Tests/polyglot-sources`. This is a Java lib specifically compiled for `Base_Tests`.

Reply:
Done